hooks/useCreateData.tsx

```diff
@@ -12,12 +13,15 @@ import handleTxError from "@/lib/handleTxError"
 import { useSheetRenderer } from "@/providers/SheetRendererProvider"
 import getTotalSupply from "@/lib/getTotalSupply"
 import usePreparePrivyWallet from "./usePrepareWallet"
+import { whitelistedAddresses } from "@/lib/whitelistAddress"
```

Comment:
are there multiple addresses here?
if yes, please rename to `whitelistedAddresses` (plural)
```suggestion
import { whitelistedAddresses } from "@/lib/whitelistedAddresses"
```